src/gsy_e/models/area/event_dispatcher.py

```diff
@@ -57,16 +57,16 @@ class AreaDispatcher:
     dicts with interarea agents for each market type.
     """
     def __init__(self, area: "Area"):
-        self._inter_area_agents: Dict[DateTime, OneSidedAgent] = {}
+        self._market_agents: Dict[DateTime, OneSidedAgent] = {}
```

Comment:
how about taking the opportunity and rename this to `self._spot_market_agents` to be consistent with the other namings?

Reply:
I've addressed this in d993287